.gitlab/.pre/create_release_qa_cards.yml

```diff
@@ -7,8 +7,8 @@ create_release_qa_cards:
     - !reference [.on_deploy_rc]
   script:
     - !reference [.setup_agent_github_app]
-    - ATLASSIAN_PASSWORD=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $JIRA_READ_API_TOKEN) || exit $?; export ATLASSIAN_PASSWORD
-    - export ATLASSIAN_USERNAME=robot-jira-agentplatform@datadoghq.com
+    - ATLASSIAN_PASSWORD=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $ATLASSIAN_READ token) || exit $?; export ATLASSIAN_PASSWORD
+    - ATLASSIAN_USERNAME=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $ATLASSIAN_READ user) || exit $?; export ATLASSIAN_USERNAME
```

Comment:
Isn't it `ATLASSIAN_WRITE`?

Reply:
it is thanks (for once this is not a typo but a missing `cherry-pick`) Thanks!